docs/protocol/extension_statistics.md

```diff
@@ -119,6 +120,14 @@ $model_stat =
   due to different batch size (for example, larger batches typically
   take longer to compute).
 
+- "memory_usage" : The memory usage detected during model loading, which may be
```

Comment:
I understand why it makes sense to report the memory usage in per model statistics. I was wondering whether we should include per model GPU/CPU consumption in our CPU/GPU metrics. 
https://github.com/triton-inference-server/server/blob/main/docs/user_guide/metrics.md#gpu-metrics

The downside is that for triton serving large number of models, this might blow-up.

Reply:
I think the memory usage at model level is not as dynamic, I think it belongs to model metadata or model config as if it is part of the model properties, but I don't think the protocol is open to adding new field and thus put in statistics.